docs/install/_linux.rst

```diff
@@ -68,9 +68,11 @@ To upgrade Kolibri on a Debian device without internet access, bring the updated
 Changing the owner of Kolibri system service
 --------------------------------------------
 
-If you installed Kolibri from the ``DEB`` file **prior to v0.10**, the new account named ``kolibri`` was created as the default user account (*owner*) of the system service. This configuration worked well in most cases, but it did not allow Kolibri the access to the local USB drives. From Kolibri v0.10 on, we have changed the system service configuration to select the default desktop user's account as the service owner.
+The "system service" is the script that runs Kolibri in the background when your system boots. It is particular to the Debian-based distribution.
 
-When you upgrade Kolibri **to v0.10 and later** using the ``DEB`` file, you will need to change the user account who owns the system service, move the data (user and content databases) and assign the owner permissions to your desktop user. To change the owner, follow these steps. 
+You may need to change the system service to run with the permissions of a different user account: Prior to v0.10, we used a ``kolibri`` user account as the owner of the system service. Since v0.10, we expect a desktop user's account for accessing USB storage.
```

Comment:
we may want to say `v0.10.0` or `v0.10.x` rather than `v0.10` to avoid confusion

Reply:
or, maybe not?

Reply:
I think the latest recommendation from @imps was `v0.10`. 
cc @laurenlichtman

Reply:
fine with me, as long as it doesn't confuse people to occasionally see it in both forms

Reply:
I see what you mean... I find `v0.10.x` a bit verbose inline (as opposed to commands that require precise filenames), but will defer to what @imps believe is best.